### PR TITLE
"PyQt5 Migrated"

### PR DESCRIPTION
**Pull request by plusminuschirag**
- Switched the UI from Tkinter to PyQT5.
- Fixed The Microphone Sound Issue and Getting control back from the Gui.(Issue #22)
- Groot Doesn't Crash Now. (Issue #21)

**Please do python base.py instead of python Gui.py. Since PyQt5 works best with Object Oriented Programming we had to dedicate it a file and Gui named fitted best.**
Installed the dependencies it shows.

**This pull is done by myself(@iamchiragsharma) and @nikhilmehta011.**

### CODE REVIEW

**Comment by omi10859** [comment]
Please add Screenshot/Gif

**Comment by plusminuschirag** [comment]
- Quit Confirmation is also Added.
- Type python base.py (Terminal).
- Groot Crashing Resolved.

issue23

File: Gui.py [hunk 1]
```diff
@@ -1,70 +1,62 @@
...
+        inp = QLabel('Input')
+        output = QLabel('Output')
 
-#this funtion is to remove output evertime
-def removeoutput():
-    global ans, i
-    if i>1:
-        blank.delete(0, END)
-        blank.insert(0, ans)
+        self.inpEdit = QLineEdit()  #Text Field for Input
```
Comment (lalitmee) [inline Gui.py]: Can we put the comments above the code lines not in the lines?

**Review by lalitmee — CHANGES_REQUESTED** [review]

**Comment by plusminuschirag** [comment]
Yes we can. i will do it tomorrow after exam, is that okay?

**Comment by lalitmee** [comment]
Yeah, sure.

**Comment by plusminuschirag** [comment]
Done. Please take a look.

**Review by techytushar — COMMENTED** [review]
You might also need to update the `requirements` file

File: Gui.py [hunk 2]
```diff
@@ -39,8 +43,9 @@ def initUI(self):
         self.setWindowIcon(QIcon('asset/img/groot.png'))
 
         #Function of speaker button linked:
-        speaker.clicked.connect(listen) #When the Speak button is clicked the listen function
-        #is triggered.
+        speaker.clicked.connect(listen)
+        #When the Speak button is clicked the listen function is triggered.
```
Comment (lalitmee) [inline Gui.py:46]: This should go above the line number `46` because this is for that line. :thinking:

**Review by lalitmee — CHANGES_REQUESTED** [review]

**Review by omi10859 — APPROVED** [review]

**Pull request merged**